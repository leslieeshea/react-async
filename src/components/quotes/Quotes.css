.Quotes {
  display: flex;
  justify-content: space-around;
  flex-wrap: wrap;
  align-items: center;

  li {
    width: 20vw;
    margin: 2em;
    padding: 2rem;

    background-color: #f0f0f0;
    text-align: center;
    list-style-type: none;
  }
}
