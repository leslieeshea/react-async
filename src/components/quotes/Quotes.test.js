import React from 'react';
import { shallow } from 'enzyme';
import Quotes from './Quotes';

describe('Quotes component', () => {
  it('renders a list of quotes', () => {
    const quotes = [
      {
        character: 'Fry',
        quote: 'This is my quote',
        image: '/image'
      },
      {
        character: 'Bender',
        quote: 'This is my quote too',
        image: '/image'
      }
    ];

    const wrapper = shallow(<Quotes quotes={quotes} />);
    expect(wrapper).toMatchSnapshot();
  });
});
