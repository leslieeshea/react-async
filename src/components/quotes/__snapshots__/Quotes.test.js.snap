// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes component renders a list of quotes 1`] = `
<ul
  className="Quotes"
>
  <li
    key="0"
  >
    <Quote
      quote={
        Object {
          "character": "Fry",
          "image": "/image",
          "quote": "This is my quote",
        }
      }
    />
  </li>
  <li
    key="1"
  >
    <Quote
      quote={
        Object {
          "character": "Bender",
          "image": "/image",
          "quote": "This is my quote too",
        }
      }
    />
  </li>
</ul>
`;
